src/Ethereum/Signer.h

```diff
@@ -36,27 +36,22 @@ class Signer {
     explicit Signer(uint256_t chainID) : chainID(std::move(chainID)) {}
```

Comment:
let's remove `chainID` from signer? we can always get it from the signing input

Reply:
done